src/datasets/main_ping_tables.md

```diff
@@ -10,8 +10,24 @@ To reduce the time for querying main ping data, we have included two new tables:
 
 ## Main Ping Sample: `telemetry.main_1pct`
 
-This table includes a 1% sample across all channels from `telemetry.main`. It includes 6 months of history.
+This table includes a 1% sample across all channels from `telemetry.main` (`sample_id = 0`).
+It includes 6 months of history.
+
+It includes an additional `subsample_id` field that is similar to `sample_id` and allows
```

Comment:
We already know people will ask how `subsample_id` is generated, so it might be best to put the algorithm here :)

Reply:
Done